restapi/error.go

```diff
@@ -35,6 +35,7 @@ var (
 	errInvalidLicense                     = errors.New("invalid license key")
 	errLicenseNotFound                    = errors.New("license not found")
 	errAvoidSelfAccountDelete             = errors.New("logged in user cannot be deleted by itself")
+	errAccessDenied                       = errors.New("Access Denied")
```

Comment:
Please make it lower case
```suggestion
	errAccessDenied                       = errors.New("access Denied")
```